en/deploy-ticdc.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Deploy TiCDC on Kubernetes
+summary: Learn how to deploy TiCDC on Kubernetes.
+category: how-to
+---
+
+# Deploy TiCDC on Kubernetes
```

Comment:
In our documentation, we use `in Kubernetes` instead of `on Kubernetes`. Please make change in this document accordingly.
```suggestion
# Deploy TiCDC in Kubernetes
```